src/components/KTable/KTable.vue

```diff
@@ -868,6 +868,13 @@ const initData = () => {
   hasInitialized.value = true
 }
 
+// Ensure `props.headers` are reactive.
+watch(() => props.headers as TableHeader[], (newVal: TableHeader[]) => {
+  if (newVal && newVal.length) {
+    tableHeaders.value = newVal
+  }
+})
```

Comment:
for this to be actually reactive, e.g. if you changed a property of an individual header, you would need a `deep` watcher.

As an example, if you changed `hidable: true` to `false`

Reply:
Good catch -- done.